api/src/main/java/org/openmrs/PatientProgram.java

```diff
@@ -36,23 +51,39 @@ public class PatientProgram extends BaseChangeableOpenmrsData implements Customi
 	// ******************
 	// Properties
 	// ******************
-	
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
Did we eventually decide to drop the sequence? `patient_program_patient_program_id`

Reply:
Because we are using GenerationType IDENTITY, we don't need to specify a sequence.